changelog.txt

```diff
@@ -14,6 +14,7 @@
 * Fix   - Fix localization issues
 * Tweak - Improve service listing readability
 * Add   - Support UPS as a carrier (beta)
+* Fix   - Carrier "disconnect modal" layout
```

Comment:
Thanks for getting a changelog note added! 1.24.0 is out already (the header date should be set now in latest `develop` now that we've merged back from `master). You can go ahead and start a `1.24.1` header and add your note there.

Reply:
Thanks for catching that, it looks like I had to update my local `develop` 👍